SPECS/kernel-alt.spec

```
@@ -1044,6 +1086,13 @@ fi
 %{python2_sitearch}/*
 
 %changelog
+* Mon Nov 14 2022 Rushikesh Jadhav <rushikesh7@gmail.com> - 4.19.265-1
+- Update patch level to 4.19.265
+
+* Thu Oct 06 2022 Samuel Verschelde <stormi-xcp@ylix.fr> - 4.19.227-4
+- Don't provide kernel Provides
+- We don't want kernel-alt to be pulled as build deps instead of main kernel packages
+
```

```suggestion
* Mon Nov 14 2022 Rushikesh Jadhav <rushikesh7@gmail.com> - 4.19.265-1
- Update patch level to 4.19.265
- Don't provide kernel Provides
- We don't want kernel-alt to be pulled as build deps instead of main kernel packages

```

Let's merge the two changelog entries, as we did not build kernel-alt-4.19.227-4.xcpng8.2

---

Agreed. Updated the commit.